.gitignore

```diff
@@ -0,0 +1,93 @@
+
+# Created by https://www.gitignore.io/api/go,vim,linux,macos
+# Edit at https://www.gitignore.io/?templates=go,vim,linux,macos
+
+### Go ###
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+
+### Go Patch ###
+/vendor/
+/Godeps/
+
```

Comment:
Remove those lines as we are keeping the vendored deps for build reliability, security and speed.